nagiosrest_plugin/tasks.py

```diff
@@ -151,7 +151,7 @@ def _make_call(ctx, request_method, url, data, operation_inputs):
             )
         )
     elif result.status_code >= 400:
-        raise NonRecoverableError(
+        raise RecoverableError(
             'Parameters passed to server were incorrect. '
```

Comment:
If this is going to be recoverable, the error should note that the server may not be up yet, e.g.
...incorrect, or Nagios is still starting.'

Reply:
Ok, fixed it - thanks !